packages/cadl-ranch-specs/http/azure/core/basic/mockapi.ts

```diff
@@ -74,14 +74,22 @@ Scenarios.Azure_Core_Basic_list = passOnSuccess(
   }),
 );
 
-Scenarios.Azure_Core_Basic_listWithPage = passOnSuccess(
+Scenarios.Azure_Core_Basic_listWithPage = passOnSuccess([
   mockapi.get("/azure/core/basic/page", (req) => {
     const responseBody = {
       value: [validUser],
+      nextLink: '/azure/core/basic/page/nextLink?%24skiptoken={"bar"%3a"foo"}',
     };
     return { status: 200, body: json(responseBody) };
   }),
-);
+  mockapi.get("/azure/core/basic/page/nextLink", (req) => {
```

Comment:
It won't break existing test since final result is same with before.

Reply:
@msyyc I think this should be a separate operation and test. I'm assuming we're explicitly testing empty next links, is that correct? If so we can call the test `emptyNextLink` or somethign like that

Reply:
Got it. Let me add a new test to check it